regression/scripts/VTOL/test_Tiltwing.py

```diff
@@ -60,7 +60,7 @@ def main():
 
     # lift Coefficient Check During Cruise
     lift_coefficient        = results.segments.cruise.conditions.aerodynamics.lift_coefficient[0][0] 
-    lift_coefficient_true   = 1.27853951187073 # 0.6482051842323242 Correct but Travis has a isssue
+    lift_coefficient_true   = 1.27853951187073 # 0.6510038496566867 Correct but Travis has a isssue
```

Comment:
What does this comments mean?

Reply:
Travis was having some issues with this but I corrected it. Issue resolved